src/main/java/com/ait/lienzo/client/core/shape/wires/util/WiresConnectorLabelFactory.java

```diff
@@ -49,18 +50,17 @@ public void accept(final Segment segment,
                     final double  sin      = Math.sin(rotation);
 
                     // Wrap the text.
-                    final double         maxDistanceX = distance > TEXT_WRAP_MAX_WIDTH ? TEXT_WRAP_MAX_WIDTH : distance;
+                    final double         maxDistanceX = TEXT_WRAP_MAX_WIDTH;
```

Comment:
why not taking into account the actual segment's `distance` (length) as for text wrapping goals? Otherwise, by setting always the constant `TEXT_WRAP_MAX_WIDTH`, it will never wrap as when the segment grows/shrinks, right? Is this due to some UX restriction?

Reply:
@romartin yeah, if you take a look on the https://issues.jboss.org/browse/JBPM-7718 the idea is to not wrap the text in case the segment is to short because it gets strange to read, so fixing the max wrap avoid always wrapping the text on segment resize,  WDYT, well it is a matter of the UX perspective, we just need to have a decision on what is the best for this case.